server/command/find_meeting_times.go

```diff
@@ -17,6 +17,9 @@ func (c *Command) findMeetings(parameters ...string) (string, bool, error) {
 	var attendees []remote.Attendee
 	for a := range parameters {
 		s := strings.Split(parameters[a], ":")
+		if len(s) != 2 {
+			return "", false, fmt.Errorf("error in parameter %s", parameters[a])
+		}
```

Comment:
`for a := range` seems weird here. We're not even using the index anywhere but the beginning. Thoughts on changing to `for _, a := range`?